client/dist/styles/relatedProducts.css

```diff
@@ -41,11 +47,58 @@
   align-self: center;
 }
 
-.actionStar {
-  justify-self: flex-end;
-  order: 5;
+/*--------------
+RP MODAL STLYING
+---------------*/
+.relatedProductsModal {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  border-radius: 5px;
```

Comment:
i have alot of px in my css too - maybe use em or rem instead?

Reply:
Yep, going to have to make that transition soon. So easy just to use pix because they are easy to imagine haha, but for mobile it will be necessary to make the shift